projects/storefrontlib/src/lib/user/components/register/register.component.scss

```diff
@@ -1,13 +1,15 @@
-@import 'theme';
-@import 'functions';
-@import 'mixins';
-//register page variables
-$cx-register-registration-link-text-decoration: underline !important;
-.cx-section__login-link {
-  margin-top: $spacer;
+:host {
+  --cx-text-decoration: underline;
```

Comment:
These shouldn't be used like this. By doing it this way, the properties are basically impossible to override. The value should be set in the fallback. So we would get something like this:

a {
 text-decoration: var(--cx-text-decoration, underline);
}

This way the fall back will be used unless a customer explicitly specifies a value for the property in his styles.scss file.

Reply:
got it, i'll review this, thx